pkg/build/netbsd.go

```diff
@@ -66,3 +73,47 @@ func (ctx netbsd) clean(kernelDir string) error {
 	// machine.
 	return nil
 }
+
+// Copy the compiled kernel to the qemu disk image using ssh
+func CopyKernelToDisk(outputDir string) error {
+	temp := []byte(`
+{
+	"qemu": "qemu-system-x86_64",
+	"cpu": 1,
+	"count": 1,
+	"mem": 1024
+}	`)
+	VMconfig := (*json.RawMessage)(&temp)
+	// Create config for booting the disk image
+	cfg := &mgrconfig.Config{
+		Workdir:      outputDir,
+		Image:        filepath.Join(outputDir, "image"),
+		SSHKey:       filepath.Join(outputDir, "key"),
+		SSHUser:      "root",
+		TargetOS:     "netbsd",
+		TargetVMArch: "amd64",
+		Type:         "qemu",
+		VM:           *VMconfig,
+	}
+	// Create a VM pool
+	pool, err := vm.Create(cfg, true)
```

Comment:
We will need to turn off debug=true once we get it working, it produces lots of output on console.